checkov/github/checks/webhooks_https_orgs.py

```diff
@@ -34,10 +34,8 @@ def scan_entity_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[Chec
                     secret = item_config.get('secret', '')
                     if re.match("^http://", url) or insecure_ssl != '0' and secret != '********':  # nosec
                         return CheckResult.FAILED, item_config
-        if org_webhooks_schema.validate(conf):
```

Comment:
Why drop schema validation?

Reply:
Hi 😃 
Originally it is done twice on the same schema. if in the first place it's the correct schema- It will be Pass/Fail. If the validation fails, the reply is not relevant for the check, and the result is unknown.